test/action_controller/json_api/linked_test.rb

```diff
@@ -137,6 +142,12 @@ def test_render_resource_with_nested_has_many_include
           assert_equal expected_linked, response['included']
         end
 
+        def test_render_resource_with_include_of_custom_key_by_original
+          get :render_resource_with_include_of_custom_key_by_original
+          response = JSON.parse(@response.body)
+          assert response.key? 'included'
```

Comment:
oh hey, this is what I found, so now we get to talk about what's supposed to happen :-)

```
  {
    "data"=>{
      "id"=>"1", "type"=>"posts", 
      "relationships"=>{
        "reviews"=>{"data"=>[{"id"=>"1", "type"=>"comments"}, {"id"=>"2", "type"=>"comments"}]},
        "writer"=>{"data"=>{"id"=>"1", "type"=>"authors"}}, 
        "site"=>{"data"=>{"id"=>"1", "type"=>"blogs"}}}}, 
    "included"=>[
      {"id"=>"1", "type"=>"comments", "attributes"=>{"body"=>"ZOMG A COMMENT"},
       "relationships"=>{"post"=>{"data"=>{"id"=>"1", "type"=>"posts"}}, "author"=>{"data"=> {"id"=>"2", "type"=>"authors"}}}}, 

      {"id"=>"2", "type"=>"comments", "attributes"=>{"body"=>"ZOMG ANOTHER COMMENT"}, "relationships"=>{"post"=>{"data"=>{"id"=>"1", "type"=>"posts"}}, "author"=>{"data"=>nil}}}]}


so, the `key` is specified in `relationships`, but the only thing related in `included` is the `type`, but I don't think that should change with `key`?